precompile/contracts/warp/contract_test.go

```diff
@@ -723,7 +733,7 @@ func TestPackEvents(t *testing.T) {
 
 	_, data, err := PackSendWarpMessageEvent(
 		sourceAddress,
-		common.Hash(unsignedMsg.ID()),
+		common.Hash(unsignedWarpMessage.ID()),
```

Comment:
Is there any way to make this test fail when it is provided the incorrect ID?

It seems like something definitely isn't being tested here.